include/variables.h

```diff
@@ -1263,7 +1263,7 @@ extern UNK_PTR D_801C02F8;
 // extern UNK_TYPE1 D_801C07F0;
 // extern UNK_TYPE1 D_801C0820;
 // extern UNK_TYPE1 D_801C0838;
-// extern UNK_TYPE1 D_801C0850;
+extern Gfx D_801C0850[];
```

Comment:
Now that objects are in, this may be documented if you wish.

Reply:
This is data from z_player_lib, not an object

Reply:
Though looking at how it is used, I'm not sure its a Gfx, It is getting placed in segment `0xC`.

Reply:
Nevermind looking at the data it is a Gfx.